rotkehlchen/externalapis/cryptocompare.py

```diff
@@ -521,23 +539,35 @@ def query_endpoint_pricehistorical(
             raise PriceQueryUnsupportedAsset(e.identifier) from e
 
         params = {
-            'fsym': cc_from_asset_symbol,
-            'tsyms': cc_to_asset_symbol,
-            'ts': timestamp,
+            'limit': 1,
```

Comment:
can you add a small comment next to limi and market attributes explaining what they are/do?